src/components/svg-canvas-graph.tsx

```diff
@@ -135,6 +139,10 @@ const SvgCanvas = () => {
                 // the node is just placed and should not trigger any save, only display the details
                 dispatch(addSelected(node));
             }
+        } else if (mode === 'select') {
+            dispatch(setMode('free'));
```

Comment:
If we propagate the pointer up event to svg-wrapper, then this `setMode('free')` could be removed as it will be set in svg-wrapper.

Reply:
That's right. I will comment e.stopPropagation in this function and remove this 'if'.